fix compilation error:
error: cannot convert 'int32x2_t' to 'int8x8_t'
see https://github.com/tensorflow/tensorflow/pull/29515
--- tensorflow/lite/build_def.bzl.orig	2021-01-21 00:25:54.000000000 +0000
+++ tensorflow/lite/build_def.bzl	2021-04-16 19:43:53.868592783 +0000
@@ -25,6 +25,10 @@
             "/DTFL_COMPILE_LIBRARY",
             "/wd4018",  # -Wno-sign-compare
         ],
+        str(Label("//tensorflow:linux_aarch64")): [
+            "-flax-vector-conversions",
+            "-fomit-frame-pointer",
+        ],
         "//conditions:default": [
             "-Wno-sign-compare",
         ],
